Added more procedures

gateslm opened the pull request.

Commit:
Added more procedures

cbudo commented:
Hey Larry,
can you sync up with the current branch before making a pull? also, please give a description with any requests, It makes it easier if we ever need to revert

Commit:
Merge remote-tracking branch 'refs/remotes/cbudo/master'

Conflicts:
	SpeedyChefAPIv2/SpeedyChef.dbml
	SpeedyChefAPIv2/SpeedyChef.designer.cs

Commit:
Fixed merge conflicts

Here is my comment. If I new how branches worked, I wouldn't have had
this problem. I used git shell and did git pull and it didn't give me
anything. So, I didn't know anyone committed anything
